library/network/src/Makefile.am

```diff
@@ -52,6 +52,11 @@ yfwdapilib_DATA = \
   lib/y2firewall/firewalld/api/services.rb \
   lib/y2firewall/firewalld/api/zones.rb
 
-EXTRA_DIST = $(module_DATA) $(scrconf_DATA) $(agent_SCRIPTS) $(ylib_DATA) $(yfwlib_DATA) $(yfwdlib_DATA) $(yfwdapilib_DATA)
+yfwhelperslibdir = @ylibdir@/y2firewall/helpers
+yfwhelperslib_DATA = \
+  lib/y2firewall/helpers/interfaces.rb
+
+EXTRA_DIST = $(module_DATA) $(scrconf_DATA) $(agent_SCRIPTS) $(ylib_DATA) \
+						 $(yfwlib_DATA) $(yfwdlib_DATA) $(yfwdapilib_DATA) $(yfwhelperslib_DATA)
```

Comment:
Indentation looks weird here.

Reply:
Yep, that was mainly because of my vim config. Will fix it